module/clock/src/mod_clock.c

```diff
@@ -366,9 +369,11 @@ static int clock_init(fwk_id_t module_id, unsigned int element_count,
         return FWK_SUCCESS;
     }
 
+#ifdef BUILD_HAS_NOTIFICATION
```

Comment:
is this change required ?

Reply:
struct mod_clock_config {
    const fwk_id_t pd_transition_notification_id;
    const fwk_id_t pd_pre_transition_notification_id;
};

Clock config is only used to set notification id and is null when notification is not supported